api/pkg/imagebuilder/imagebuilder.go

```diff
@@ -43,6 +45,27 @@ import (
 
 var maxCheckImageRetry uint64 = 2
 
+var (
+	imageBuilderAttempts = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "merlin",
+		Name:      "image_builder_attempts",
+		Help:      "Total of image builder attempts",
+	}, []string{"project", "model", "model_type", "model_version", "python_version"})
+
+	imageBuilderResults = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
The histogram metric below comes with a counter as well (`merlin_image_builder_duration_seconds_count`), right? Perhaps we don't need this separate results metric.

Reply:
This is good suggestion but I prefer to explicitly have results metrics instead of bury the result information in `merlin_image_builder_duration_seconds_count` metrics. `merlin_image_builder_results` as metrics name is easier to find and understand.